packages/skill-lesson/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@skillrecordings/skill-lesson",
```

Comment:
this moves all of the "logic bits" into `skill-lesson` package leaving the UI components behind